src/plugins/console/README.md

```diff
@@ -234,3 +234,43 @@ Some autocomplete definitions need to be configured with dynamic values that can
 A list of variables is defined in the  `parametrizedComponentFactories` function in [`kb.js`](https://github.com/elastic/kibana/blob/main/src/plugins/console/public/lib/kb/kb.js) file. The values of these variables are assigned dynamically for every cluster. 
 Use these variables with curly braces, for example `{indices}`, `{types}`, `{id}`, `{username}`, `{template}`, `{nodes}` etc.
 
+### New features
```

Comment:
[nit] I think we can probably move these as part of the `##Features` section that already exists above in the readme.